src/util/formDevices.tsx

```diff
@@ -75,16 +75,24 @@ export const parseDevices = (devices: LxdDevices): FormDevice[] => {
         (key) => !["type", "name", "network"].includes(key)
       );
 
+    if (isCustomNetwork) {
+      return {
+        name: key,
+        bare: item,
+        type: "custom-nic",
+      };
+    }
+
+    if (key === "iso-volume") {
+      return {
+        name: key,
+        bare: item as never,
```

Comment:
`item` seems to have a value here, shouldn't this be treated as `unknown`?
```suggestion
        bare: item as unknown,
```

Reply:
Right, `unknown` is the better choice here. I also had to change the `UnknownDevice` type for this to work though.